cmd/kuberhealthy/reaper.go

```diff
@@ -89,11 +83,11 @@ func reaper(ctx context.Context) {
 
 	log.Infoln("checkReaper: starting up...")
 	log.Infoln("checkReaper: job cleanup duration:", cfg.JobCleanupDuration)
-	log.Infoln("checkReaper: max checker pods:", cfg.MaxCheckPods)
+	log.Infoln("checkReaper: max completed checker pods:", cfg.MaxCompletedCheckPods)
 	log.Infoln("checkReaper: failed pod cleanup duration:", cfg.FailedPodCleanupDuration)
 
 	// start a new ticker
-	t := time.NewTicker(time.Minute * 3)
+	t := time.NewTicker(time.Second * 30)
```

Comment:
Let's make this configurable from an environment variable and default it to `30s` if not configured.  How about the env var `CHECK_REAPER_RUN_INTERVAL` with type of `time.Duration`?

Reply:
feel like it would have to go in the kuberhealthy configmap no? that's where the other checkReaper configurations are being set.

Reply:
actually, you're right, setting it as an env var is better.